app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -53,17 +53,16 @@
 
           <TextView
             android:id="@+id/chapter_title"
+            style="@style/Heading3"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
```

Comment:
layout_height can be removed from this and other files as it is `wrap_content`

Reply:
Ditto for other files.

Reply:
Done.